data_store/transformation/towns_fund/tf_transform_r3.py

```diff
@@ -352,7 +352,7 @@ def extract_project_progress(df_data: pd.DataFrame, project_lookup: dict, report
     :return: A new DataFrame containing the extracted project progress rows.
     """
     # if round 4 or 5, ingest two additional columns
-    df_data = df_data.iloc[18:39, 2:15] if reporting_round in (4, 5) else df_data.iloc[18:39, 2:13]
+    df_data = df_data.iloc[18:39, 2:15] if reporting_round >= 4 else df_data.iloc[18:39, 2:13]
```

Comment:
@wjrm500  - thoughts about this or should we be duplicating into a new r6?

Reply:
Have already discussed this in the meeting this afternoon but just to get my thoughts down briefly on paper - reckon it's okay like this, the question of how much code we want to share / duplicate across rounds, and how we handle reporting round transition / deprecation is an interesting one, but I guess my overriding thought here is just to minimise code changes related to TF